test/test_configuration_space.py

```diff
@@ -803,6 +804,24 @@ def test_acts_as_mapping(self):
         assert list(d.items()) == list(zip(names, hyperparameters))
         assert len(d) == 5
 
+    def test_remove_parameter_priors(self):
+        cs = ConfigurationSpace()
+        cs.add_hyperparameter(UniformIntegerHyperparameter('integer', 1, 5, log=True))
+        cs.add_hyperparameter(CategoricalHyperparameter('cat', [0, 1, 2], weights=[1, 2, 3]))
+        cs.add_hyperparameter(BetaFloatHyperparameter("beta", alpha=8, beta=2, lower=-1, upper=11))
+        cs.add_hyperparameter(NormalIntegerHyperparameter("norm", mu=5, sigma=4, lower=1, upper=15))
+
+        uniform_cs = cs.remove_parameter_priors()
+        
+        expected_cs = ConfigurationSpace()
+        expected_cs.add_hyperparameter(UniformIntegerHyperparameter('integer', 1, 5, log=True))
+        expected_cs.add_hyperparameter(CategoricalHyperparameter('cat', [0, 1, 2]))
+        # These default values are computed closed form for beta, trivial for normal
```

Comment:
The default is the mode of the distribution for all distributionss (if nothing else is given as default), so that could be said at the very least! 

I'll add what you proposed.